var/spack/repos/builtin/packages/py-prompt-toolkit/package.py

```diff
@@ -11,6 +11,20 @@ class PyPromptToolkit(PythonPackage):
 
     pypi = "prompt_toolkit/prompt_toolkit-1.0.9.tar.gz"
 
+    # 'prompt_toolkit.contrib.ssh' requires 'asyncssh'
```

Comment:
Please clarify the tie between the comment and the listed modules.  Perhaps a link?

Reply:
The `import_modules` list is the list of all detected modules minus `prompt_toolkit.contrib.ssh`, which requires an additional optional dependency to be imported.

Reply:
I see. Perhaps add to the comment something along the lines that the module is being excluded because of the dependency?

Reply:
I'm not sure if I've ever seen a single person other than myself use `import_modules`, so this comment is really only for my own posterity...

Reply:
It also only applies to `PythonPackage`, of which I'm the listed `maintainer`